tests/model_fields/test_mixins.py

```diff
@@ -0,0 +1,69 @@
+from django.db.models.fields.mixins import FieldCacheMixin
+from django.test import SimpleTestCase
+from django.utils.deprecation import RemovedInDjango60Warning
+from django.utils.functional import cached_property
+
+from .models import Foo
+
+
+class ExampleOld(FieldCacheMixin):
```

Comment:
These are great tests, thank you for adding them. But, I think we should add some `RemovedInDjango60Warning` comments through this file to ensure we do the proper cleanup also at the test level when we bootstrap Django 6.0. Would you agree?